### PR TITLE
[Merged by Bors] - chore: some renames `iff_open` -> `iff_isOpen`

### PR DESCRIPTION
**Pull request by grunweg**
Our naming convention prefers `isOpen`. See the individual commit messages for details.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by jcommelin — COMMENTED** [review]
Thanks :tada: 

If CI passes, please remove the label `awaiting-CI` and merge this yourself, by adding a comment `bors r+`.

bors d+

File: Mathlib/Topology/Defs/Filter.lean [hunk 1]
```diff
@@ -202,8 +202,8 @@ infixl:300 " ⤳ " => Specializes
 equivalent properties hold:
 
 - `𝓝 x = 𝓝 y`; we use this property as the definition;
-- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_open`;
-- for any closed set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_closed`;
+- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `Inseparable.mem_open_iff`;
```
Comment (fpvandoorn) [inline Mathlib/Topology/Defs/Filter.lean:205]:
```suggestion
- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_forall_open`;
```
We're claiming to point at the equivalence to inseparable, don't we?

Comment (grunweg) [reply Mathlib/Topology/Defs/Filter.lean]: Yes, indeed! Fixed, also for closedness.

File: Mathlib/Topology/Defs/Filter.lean [hunk 2]
```diff
@@ -202,8 +202,8 @@ infixl:300 " ⤳ " => Specializes
 equivalent properties hold:
 
 - `𝓝 x = 𝓝 y`; we use this property as the definition;
-- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_open`;
-- for any closed set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_closed`;
+- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `Inseparable.iff_forall_open`;
```
Comment (fpvandoorn) [inline Mathlib/Topology/Defs/Filter.lean:205]:
```suggestion
- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_forall_open`;
```

**Comment by grunweg** [comment]
bors merge

**Comment by grunweg** [comment]
bors merge

**Pull request closed without merging**